### PR TITLE
Replaced pipes with spaces

### PR DESCRIPTION
**Pull request by danielquinn**
Looking for code review on this one.  Much of what I've changed is just PyCharm's linter complaining, and a few typos, but specifically, there's one spot where I've changed the use of pipes to spaces in the probe output to be more like other common unix command line tools (`df`, `ls`, etc.)

### CODE REVIEW

File: ripe/atlas/tools/commands/probes.py [hunk 1]
```diff
@@ -142,17 +146,16 @@ def run(self):
         self.renderer.on_start()
 
         if self.arguments.aggregate_by:
-
             aggregators = self.get_aggregators()
             buckets = aggregate(probes, aggregators)
             self.renderer.render_aggregation(buckets)
         else:
-
             self.renderer.on_table_title()
```
Comment (astrikos) [inline ripe/atlas/tools/commands/probes.py]: I like to think that with a new line is easier to digest, but maybe it's me :)

File: ripe/atlas/tools/renderers/probes.py [hunk 2]
```diff
@@ -52,13 +54,13 @@ def _construct_probe_line(self):
...
-            self.probe_template = "{:<5}|{:<6}|{:<6}|{:^2}|{:<}|"
+            self.probe_template = "{:<5} {:<6} {:<6} {:^2} {:<}"
         else:
             for field in self.fields:
-                self.probe_template += "{:<}|"
+                self.probe_template += " {:<}"
 
         for fields in self.additional_fields:
-            self.probe_template += "{:<}|"
+            self.probe_template += " {:<}"
```
Comment (astrikos) [inline ripe/atlas/tools/renderers/probes.py]: All the above probably has broken tests, could you run them and see if it's okay, if not change tests as well to depict new format

**Comment by astrikos** [comment]
Seems good to me, if you make tests pass we can merge.

File: ripe/atlas/tools/commands/probes.py (hunk 1 above)
Comment (danielquinn) [inline ripe/atlas/tools/commands/probes.py]: I like the new line, but then I would put an additional new line above line 152 as well to keep things symetrical.  I can undo this bit though if it makes you sad.

File: ripe/atlas/tools/renderers/probes.py (hunk 2 above)
Comment (danielquinn) [inline ripe/atlas/tools/renderers/probes.py]: Oh I'm sure it breaks everything, but I didn't want to bother tweaking the tests if you weren't cool with the changes.

File: ripe/atlas/tools/commands/probes.py (hunk 1 above)
Comment (astrikos) [inline ripe/atlas/tools/commands/probes.py]: keep the new lines and add the the new line above 152 to keep it symetrical :)

File: ripe/atlas/tools/renderers/probes.py (hunk 2 above)
Comment (astrikos) [inline ripe/atlas/tools/renderers/probes.py]: Sure, go ahead

**Comment by danielquinn** [comment]
I've just updated the tests to allow for a space-separated grid.

File: ripe/atlas/tools/probes/__init__.py [hunk 3]
```diff
@@ -11,10 +11,6 @@ class Probe(object):
 
     EXPIRE_TIME = 60 * 60 * 24 * 30
 
-    def __init__(self, id, meta_data=None):
-
-        return CProbe(id, meta_data)
-
```
Comment (astrikos) [inline ripe/atlas/tools/probes/__init__.py]: Why? It's useful if you want to get probe without any cache! It's free, so I would let it live there

Comment (danielquinn) [inline ripe/atlas/tools/probes/__init__.py]: PyCharm highlighted it as an error, so I dug into it:

    Cannot return a value from __init__:
    This inspection reports occurrences of return statements with a return value inside
    __init__ methods of classes. A constructor should not return any value.

it also said this:

    This inspection reports discrepancies between declared parameters and
    actual arguments, as well as incorrect arguments (e.g. duplicate named
    arguments) and incorrect argument order. Decorators are analyzed, too.

I think what you may be looking for is `__new__()` or better yet, something like `get()`?

Comment (astrikos) [inline ripe/atlas/tools/probes/__init__.py]: Anyway, we don't use this feature for now, so it's okay that you delete it. If you need it we can come back to it. I am merging it :)

**Pull request merged**